### PR TITLE
Implement universal reusable Knowledge base component

### PR DESCRIPTION
**Pull request by titkov49**

### CODE REVIEW

**Review by ateska — APPROVED** [review]
As discussed on the review, this is OK ... but we will likely need to deal with absolute paths in the `index.json`.

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Can you please add changes into CHANGELOG file? Also some printscreens would be cool

**Comment by ateska** [comment]
A little bit of documentation (e.g. example of use) would be nice here.

**Comment by titkov49** [comment]
@Pe5h4 re-review please :)

File: doc/knowledge-base.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+```
+import React from 'react';
+
+import { KnowledgeBase } from 'asab-webui';
+
+const KnowledgeBaseContainer = (props) => {
+	return (
+		<KnowledgeBase
+			app={props.app}
+			api="content"
```
Comment (Pe5h4) [inline doc/knowledge-base.md:121]: @titkov49 `api` might be misleading here. I would recommend to use something like `apiPath` or something with informative value

Comment (titkov49) [reply doc/knowledge-base.md]: @Pe5h4 `api` changed to `apiPath`

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**